### ISSUE TITLE
3.0.16: test suite is failing in 'icalrecurtest-r` unit

### ISSUE THREAD

**Issue by kloczek** [libical/libical#635]
cmake setup (`cmake -L`)
```console
-- Cache values
CMAKE_BUILD_TYPE:STRING=RelWithDebInfo
CMAKE_INSTALL_PREFIX:PATH=/usr
ENABLE_GTK_DOC:BOOL=ON
GOBJECT_INTROSPECTION:BOOL=ON
GTKDOC_FIXXREF:FILEPATH=/usr/bin/gtkdoc-fixxref
GTKDOC_MKDB:FILEPATH=/usr/bin/gtkdoc-mkdb
GTKDOC_MKHTML:FILEPATH=/usr/bin/gtkdoc-mkhtml
GTKDOC_SCAN:FILEPATH=/usr/bin/gtkdoc-scan
GTKDOC_SCANGOBJ:FILEPATH=/usr/bin/gtkdoc-scangobj
ICAL_ALLOW_EMPTY_PROPERTIES:BOOL=ON
ICAL_BUILD_DOCS:BOOL=ON
ICAL_ERRORS_ARE_FATAL:BOOL=OFF
ICAL_GLIB:BOOL=ON
ICAL_GLIB_VAPI:BOOL=ON
ICU_DERB_EXECUTABLE:FILEPATH=ICU_DERB_EXECUTABLE-NOTFOUND
ICU_GENBRK_EXECUTABLE:FILEPATH=ICU_GENBRK_EXECUTABLE-NOTFOUND
ICU_GENCCODE_EXECUTABLE:FILEPATH=ICU_GENCCODE_EXECUTABLE-NOTFOUND
ICU_GENCFU_EXECUTABLE:FILEPATH=ICU_GENCFU_EXECUTABLE-NOTFOUND
ICU_GENCMN_EXECUTABLE:FILEPATH=ICU_GENCMN_EXECUTABLE-NOTFOUND
ICU_GENCNVAL_EXECUTABLE:FILEPATH=ICU_GENCNVAL_EXECUTABLE-NOTFOUND
ICU_GENDICT_EXECUTABLE:FILEPATH=ICU_GENDICT_EXECUTABLE-NOTFOUND
ICU_GENNORM2_EXECUTABLE:FILEPATH=ICU_GENNORM2_EXECUTABLE-NOTFOUND
ICU_GENRB_EXECUTABLE:FILEPATH=ICU_GENRB_EXECUTABLE-NOTFOUND
ICU_GENSPREP_EXECUTABLE:FILEPATH=ICU_GENSPREP_EXECUTABLE-NOTFOUND
ICU_ICU-CONFIG_EXECUTABLE:FILEPATH=/usr/bin/icu-config
ICU_ICUINFO_EXECUTABLE:FILEPATH=/usr/bin/icuinfo
ICU_ICUPKG_EXECUTABLE:FILEPATH=ICU_ICUPKG_EXECUTABLE-NOTFOUND
ICU_INCLUDE_DIR:PATH=/usr/include
ICU_MAKECONV_EXECUTABLE:FILEPATH=ICU_MAKECONV_EXECUTABLE-NOTFOUND
ICU_MAKEFILE_INC:FILEPATH=/usr/lib64/icu/72.1/Makefile.inc
ICU_PKGDATA_EXECUTABLE:FILEPATH=ICU_PKGDATA_EXECUTABLE-NOTFOUND
ICU_PKGDATA_INC:FILEPATH=/usr/lib64/icu/72.1/pkgdata.inc
ICU_UCONV_EXECUTABLE:FILEPATH=ICU_UCONV_EXECUTABLE-NOTFOUND
INCLUDE_INSTALL_DIR:STRING=include
LIBICAL_BUILD_TESTING:BOOL=ON
LIB_INSTALL_DIR:STRING=lib64
PYTHON3:FILEPATH=/usr/bin/python3
SHARED_ONLY:BOOL=ON
SHARE_INSTALL_DIR:STRING=share
STATIC_ONLY:BOOL=OFF
VALAC:FILEPATH=/usr/bin/valac
VAPIGEN:FILEPATH=/usr/bin/vapigen
WITH_CXX_BINDINGS:BOOL=ON
```
and test suite is failing:
```console
+ cd libical-3.0.16
+ /usr/bin/ctest --test-dir x86_64-redhat-linux-gnu --output-on-failure --force-new-ctest-process -j48
Internal ctest changing into directory: /home/tkloczko/rpmbuild/BUILD/libical-3.0.16/x86_64-redhat-linux-gnu
Test project /home/tkloczko/rpmbuild/BUILD/libical-3.0.16/x86_64-redhat-linux-gnu
      Start  1: regression
      Start  2: parser-0
      Start  3: parser-1-1
      Start  4: parser-1
      Start  5: parser-2
      Start  6: parser-2445
      Start  7: parser-2446
      Start  8: parser-3
      Start  9: parser-4
      Start 10: parser-5
      Start 11: parser-6
      Start 12: parser-7
      Start 13: parser-8
      Start 14: parser-calendar
      Start 15: parser-caltime
      Start 16: parser-classify
      Start 17: parser-crash
      Start 18: parser-get_char_test
      Start 19: parser-incoming
      Start 20: parser-issue250
      Start 21: parser-issue251
      Start 22: parser-issue252
      Start 23: parser-issue253
      Start 24: parser-issue339
      Start 25: parser-large
      Start 26: parser-malloc
      Start 27: parser-overlaps
      Start 28: parser-process-calendar
      Start 29: parser-process-incoming
      Start 30: parser-recur-errors
      Start 31: parser-restriction
      Start 32: parser-smallcluster
      Start 33: parser-spanlist
      Start 34: parser-stresstest
      Start 35: parser-zday
      Start 36: recur
      Start 37: icaltm_test
      Start 38: testmime
      Start 39: testvcal
      Start 40: process
      Start 41: timezones
      Start 42: builtin_timezones
      Start 43: icalrecurtest-r
      Start 44: icalrecurtest
      Start 45: libical-glib-array
      Start 46: libical-glib-attach
      Start 47: libical-glib-component
      Start 48: libical-glib-comprehensive
 1/57 Test  #2: parser-0 .........................   Passed    0.07 sec
      Start 49: libical-glib-duration
 2/57 Test  #3: parser-1-1 .......................   Passed    0.07 sec
      Start 50: libical-glib-error
 3/57 Test  #4: parser-1 .........................   Passed    0.07 sec
      Start 51: libical-glib-misc
 4/57 Test  #5: parser-2 .........................   Passed    0.07 sec
      Start 52: libical-glib-parameter
 5/57 Test  #6: parser-2445 ......................   Passed    0.07 sec
      Start 53: libical-glib-period
 6/57 Test  #7: parser-2446 ......................   Passed    0.07 sec
      Start 54: libical-glib-property
 7/57 Test  #8: parser-3 .........................   Passed    0.07 sec
      Start 55: libical-glib-recurrence
 8/57 Test  #9: parser-4 .........................   Passed    0.08 sec
      Start 56: libical-glib-timezone
 9/57 Test #10: parser-5 .........................   Passed    0.08 sec
      Start 57: libical-glib-value
10/57 Test #11: parser-6 .........................   Passed    0.08 sec
11/57 Test #12: parser-7 .........................   Passed    0.07 sec
12/57 Test #13: parser-8 .........................   Passed    0.07 sec
13/57 Test #14: parser-calendar ..................   Passed    0.07 sec
14/57 Test #15: parser-caltime ...................   Passed    0.07 sec
15/57 Test #16: parser-classify ..................   Passed    0.07 sec
16/57 Test #17: parser-crash .....................   Passed    0.07 sec
17/57 Test #18: parser-get_char_test .............   Passed    0.07 sec
18/57 Test #19: parser-incoming ..................   Passed    0.07 sec
19/57 Test #20: parser-issue250 ..................   Passed    0.06 sec
20/57 Test #21: parser-issue251 ..................   Passed    0.06 sec
21/57 Test #22: parser-issue252 ..................   Passed    0.06 sec
22/57 Test #23: parser-issue253 ..................   Passed    0.06 sec
23/57 Test #24: parser-issue339 ..................   Passed    0.06 sec
24/57 Test #26: parser-malloc ....................   Passed    0.06 sec
25/57 Test #27: parser-overlaps ..................   Passed    0.06 sec
26/57 Test #28: parser-process-calendar ..........   Passed    0.06 sec
27/57 Test #29: parser-process-incoming ..........   Passed    0.06 sec
28/57 Test #30: parser-recur-errors ..............   Passed    0.05 sec
29/57 Test #31: parser-restriction ...............   Passed    0.05 sec
30/57 Test #32: parser-smallcluster ..............   Passed    0.05 sec
31/57 Test #33: parser-spanlist ..................   Passed    0.05 sec
32/57 Test #34: parser-stresstest ................   Passed    0.05 sec
33/57 Test #35: parser-zday ......................   Passed    0.05 sec
34/57 Test #38: testmime .........................   Passed    0.05 sec
35/57 Test #39: testvcal .........................   Passed    0.04 sec
36/57 Test #40: process ..........................   Passed    0.05 sec
37/57 Test #44: icalrecurtest ....................   Passed    0.05 sec
38/57 Test #36: recur ............................   Passed    0.07 sec
39/57 Test #46: libical-glib-attach ..............   Passed    0.08 sec
40/57 Test #45: libical-glib-array ...............   Passed    0.08 sec
41/57 Test #48: libical-glib-comprehensive .......   Passed    0.08 sec
42/57 Test #47: libical-glib-component ...........   Passed    0.08 sec
43/57 Test #49: libical-glib-duration ............   Passed    0.07 sec
44/57 Test #50: libical-glib-error ...............   Passed    0.08 sec
45/57 Test #51: libical-glib-misc ................   Passed    0.08 sec
46/57 Test #52: libical-glib-parameter ...........   Passed    0.08 sec
47/57 Test #53: libical-glib-period ..............   Passed    0.08 sec
48/57 Test #54: libical-glib-property ............   Passed    0.08 sec
49/57 Test #57: libical-glib-value ...............   Passed    0.07 sec
50/57 Test #55: libical-glib-recurrence ..........   Passed    0.08 sec
51/57 Test #56: libical-glib-timezone ............   Passed    0.09 sec
52/57 Test #25: parser-large .....................   Passed    0.15 sec
53/57 Test #37: icaltm_test ......................   Passed    0.16 sec
54/57 Test #43: icalrecurtest-r ..................***Failed    0.19 sec
CMake Error at /home/tkloczko/rpmbuild/BUILD/libical-3.0.16/cmake/run_test.cmake:43 (message):
  Output does not match for
  /home/tkloczko/rpmbuild/BUILD/libical-3.0.16/src/test/icalrecur_withicu_dangi_test.out
  and
  /home/tkloczko/rpmbuild/BUILD/libical-3.0.16/x86_64-redhat-linux-gnu/bin/test.out:
  : shell output: 1!



55/57 Test  #1: regression .......................   Passed    0.38 sec
56/57 Test #41: timezones ........................   Passed    0.51 sec
57/57 Test #42: builtin_timezones ................   Passed    7.99 sec

98% tests passed, 1 tests failed out of 57

Total Test time (real) =   8.07 sec

The following tests FAILED:
         43 - icalrecurtest-r (Failed)
Errors while running CTest
```

**Comment by kloczek** [comment]
Gentl ping .. 🤔 
Diff between files
<details>

```console
--- /home/tkloczko/rpmbuild/BUILD/libical-3.0.16/x86_64-redhat-linux-gnu/bin/test.out   2023-04-04 19:46:14.899279923 +0000
+++ /home/tkloczko/rpmbuild/BUILD/libical-3.0.16/src/test/icalrecur_withicu_dangi_test.out      2022-10-17 20:53:57.000000000 +0000
@@ -135,211 +135,3 @@
 RRULE:RSCALE=GREGORIAN;FREQ=YEARLY;BYYEARDAY=-366;SKIP=BACKWARD;COUNT=9
 DTSTART:20120101T120000
 INSTANCES:20120101T120000,20121231T120000,20131231T120000,20141231T120000,20160101T120000,20161231T120000,20171231T120000,20181231T120000,20200101T120000
-000,19971231T090000,19980101T090000,19980131T090000,19980201T090000
-
-RRULE:FREQ=MONTHLY;INTERVAL=18;COUNT=10;BYMONTHDAY=10,11,12,13,14,15
-DTSTART:19970910T090000
-INSTANCES:19970910T090000,19970911T090000,19970912T090000,19970913T090000,19970914T090000,19970915T090000,19990310T090000,19990311T090000,19990312T090000,19990313T090000
-
-RRULE:FREQ=MONTHLY;INTERVAL=2;BYDAY=TU;COUNT=18
-DTSTART:19970902T090000
-INSTANCES:19970902T090000,19970909T090000,19970916T090000,19970923T090000,19970930T090000,19971104T090000,19971111T090000,19971118T090000,19971125T090000,19980106T090000,19980113T090000,19980120T090000,19980127T090000,19980303T090000,19980310T090000,19980317T090000,19980324T090000,19980331T090000
-
-RRULE:FREQ=MONTHLY;BYDAY=FR;BYMONTHDAY=13;COUNT=5
-DTSTART:19970902T090000
-INSTANCES:19980213T090000,19980313T090000,19981113T090000,19990813T090000,20001013T090000
-
-RRULE:FREQ=MONTHLY;BYDAY=SA;BYMONTHDAY=7,8,9,10,11,12,13;COUNT=10
-DTSTART:19970913T090000
-INSTANCES:19970913T090000,19971011T090000,19971108T090000,19971213T090000,19980110T090000,19980207T090000,19980307T090000,19980411T090000,19980509T090000,19980613T090000
-
-RRULE:FREQ=MONTHLY;COUNT=3;BYDAY=TU,WE,TH;BYSETPOS=3
-DTSTART:19970904T090000
-INSTANCES:19970904T090000,19971007T090000,19971106T090000
-
-RRULE:FREQ=MONTHLY;BYDAY=MO,TU,WE,TH,FR;BYSETPOS=-2;COUNT=7
-DTSTART:19970929T090000
-INSTANCES:19970929T090000,19971030T090000,19971127T090000,19971230T090000,19980129T090000,19980226T090000,19980330T090000
-
-RRULE:FREQ=MONTHLY;BYMONTHDAY=15,30;COUNT=5
-DTSTART:20070115T090000
-INSTANCES:20070115T090000,20070130T090000,20070215T090000,20070315T090000,20070330T090000
-
-RRULE:FREQ=MONTHLY;COUNT=12
-DTSTART:20070131T090000
-INSTANCES:20070131T090000,20070331T090000,20070531T090000,20070731T090000,20070831T090000,20071031T090000,20071231T090000,20080131T090000,20080331T090000,20080531T090000,20080731T090000,20080831T090000
-
-RRULE:FREQ=WEEKLY;COUNT=10
-DTSTART:19970902T090000
-INSTANCES:19970902T090000,19970909T090000,19970916T090000,19970923T090000,19970930T090000,19971007T090000,19971014T090000,19971021T090000,19971028T090000,19971104T090000
-
-RRULE:FREQ=WEEKLY;UNTIL=19971224T000000Z
-DTSTART:19970902T090000
-INSTANCES:19970902T090000,19970909T090000,19970916T090000,19970923T090000,19970930T090000,19971007T090000,19971014T090000,19971021T090000,19971028T090000,19971104T090000,19971111T090000,19971118T090000,19971125T090000,19971202T090000,19971209T090000,19971216T090000,19971223T090000
-
-RRULE:FREQ=WEEKLY;INTERVAL=2;WKST=SU;COUNT=13
-DTSTART:19970902T090000
-INSTANCES:19970902T090000,19970916T090000,19970930T090000,19971014T090000,19971028T090000,19971111T090000,19971125T090000,19971209T090000,19971223T090000,19980106T090000,19980120T090000,19980203T090000,19980217T090000
-
-RRULE:FREQ=WEEKLY;UNTIL=19971007T000000Z;WKST=SU;BYDAY=TU,TH
-DTSTART:19970902T090000
-INSTANCES:19970902T090000,19970904T090000,19970909T090000,19970911T090000,19970916T090000,19970918T090000,19970923T090000,19970925T090000,19970930T090000,19971002T090000
-
-RRULE:FREQ=WEEKLY;COUNT=10;WKST=SU;BYDAY=TU,TH
-DTSTART:19970902T090000
-INSTANCES:19970902T090000,19970904T090000,19970909T090000,19970911T090000,19970916T090000,19970918T090000,19970923T090000,19970925T090000,19970930T090000,19971002T090000
-
-RRULE:FREQ=WEEKLY;INTERVAL=4;BYDAY=SU;COUNT=4
-DTSTART:20150322T000000
-INSTANCES:20150322T000000,20150419T000000,20150517T000000,20150614T000000
-
-RRULE:FREQ=WEEKLY;INTERVAL=2;UNTIL=19971224T000000Z;WKST=SU;BYDAY=MO,WE,FR
-DTSTART:19970901T090000
-INSTANCES:19970901T090000,19970903T090000,19970905T090000,19970915T090000,19970917T090000,19970919T090000,19970929T090000,19971001T090000,19971003T090000,19971013T090000,19971015T090000,19971017T090000,19971027T090000,19971029T090000,19971031T090000,19971110T090000,19971112T090000,19971114T090000,19971124T090000,19971126T090000,19971128T090000,19971208T090000,19971210T090000,19971212T090000,19971222T090000
-
-RRULE:FREQ=WEEKLY;INTERVAL=2;COUNT=8;WKST=SU;BYDAY=TU,TH
-DTSTART:19970902T090000
-INSTANCES:19970902T090000,19970904T090000,19970916T090000,19970918T090000,19970930T090000,19971002T090000,19971014T090000,19971016T090000
-
-RRULE:FREQ=WEEKLY;INTERVAL=2;COUNT=4;BYDAY=TU,SU;WKST=MO
-DTSTART:19970805T090000
-INSTANCES:19970805T090000,19970810T090000,19970819T090000,19970824T090000
-
-RRULE:FREQ=WEEKLY;INTERVAL=2;COUNT=4;BYDAY=TU,SU;WKST=SU
-DTSTART:19970805T090000
-INSTANCES:19970805T090000,19970817T090000,19970819T090000,19970831T090000
-
-RRULE:FREQ=WEEKLY;INTERVAL=1;COUNT=2;BYDAY=MO;
-DTSTART:20141006T090000
-INSTANCES:20141006T090000,20141013T090000
-
-RRULE:FREQ=DAILY;COUNT=10
-DTSTART:19970902T090000
-INSTANCES:19970902T090000,19970903T090000,19970904T090000,19970905T090000,19970906T090000,19970907T090000,19970908T090000,19970909T090000,19970910T090000,19970911T090000
-
-RRULE:FREQ=DAILY;UNTIL=19971224T000000Z
-DTSTART:19970902T090000
-INSTANCES:19970902T090000,19970903T090000,19970904T090000,19970905T090000,19970906T090000,19970907T090000,19970908T090000,19970909T090000,19970910T090000,19970911T090000,19970912T090000,19970913T090000,19970914T090000,19970915T090000,19970916T090000,19970917T090000,19970918T090000,19970919T090000,19970920T090000,19970921T090000,19970922T090000,19970923T090000,19970924T090000,19970925T090000,19970926T090000,19970927T090000,19970928T090000,19970929T090000,19970930T090000,19971001T090000,19971002T090000,19971003T090000,19971004T090000,19971005T090000,19971006T090000,19971007T090000,19971008T090000,19971009T090000,19971010T090000,19971011T090000,19971012T090000,19971013T090000,19971014T090000,19971015T090000,19971016T090000,19971017T090000,19971018T090000,19971019T090000,19971020T090000,19971021T090000,19971022T090000,19971023T090000,19971024T090000,19971025T090000,19971026T090000,19971027T090000,19971028T090000,19971029T090000,19971030T090000,19971031T090000,19971101T090000,19971102T090000,19971103T090000,19971104T090000,19971105T090000,19971106T090000,19971107T090000,19971108T090000,19971109T090000,19971110T090000,19971111T090000,19971112T090000,19971113T090000,19971114T090000,19971115T090000,19971116T090000,19971117T090000,19971118T090000,19971119T090000,19971120T090000,19971121T090000,19971122T090000,19971123T090000,19971124T090000,19971125T090000,19971126T090000,19971127T090000,19971128T090000,19971129T090000,19971130T090000,19971201T090000,19971202T090000,19971203T090000,19971204T090000,19971205T090000,19971206T090000,19971207T090000,19971208T090000,19971209T090000,19971210T090000,19971211T090000,19971212T090000,19971213T090000,19971214T090000,19971215T090000,19971216T090000,19971217T090000,19971218T090000,19971219T090000,19971220T090000,19971221T090000,19971222T090000,19971223T090000
-
-RRULE:FREQ=DAILY;INTERVAL=2;COUNT=20
-DTSTART:19970902T090000
-INSTANCES:19970902T090000,19970904T090000,19970906T090000,19970908T090000,19970910T090000,19970912T090000,19970914T090000,19970916T090000,19970918T090000,19970920T090000,19970922T090000,19970924T090000,19970926T090000,19970928T090000,19970930T090000,19971002T090000,19971004T090000,19971006T090000,19971008T090000,19971010T090000
-
-RRULE:FREQ=DAILY;INTERVAL=10;COUNT=5
-DTSTART:19970902T090000
-INSTANCES:19970902T090000,19970912T090000,19970922T090000,19971002T090000,19971012T090000
-
-RRULE:FREQ=DAILY;UNTIL=20000131T140000Z;BYMONTH=1
-DTSTART:19980101T090000
-INSTANCES:19980101T090000,19980102T090000,19980103T090000,19980104T090000,19980105T090000,19980106T090000,19980107T090000,19980108T090000,19980109T090000,19980110T090000,19980111T090000,19980112T090000,19980113T090000,19980114T090000,19980115T090000,19980116T090000,19980117T090000,19980118T090000,19980119T090000,19980120T090000,19980121T090000,19980122T090000,19980123T090000,19980124T090000,19980125T090000,19980126T090000,19980127T090000,19980128T090000,19980129T090000,19980130T090000,19980131T090000,19990101T090000,19990102T090000,19990103T090000,19990104T090000,19990105T090000,19990106T090000,19990107T090000,19990108T090000,19990109T090000,19990110T090000,19990111T090000,19990112T090000,19990113T090000,19990114T090000,19990115T090000,19990116T090000,19990117T090000,19990118T090000,19990119T090000,19990120T090000,19990121T090000,19990122T090000,19990123T090000,19990124T090000,19990125T090000,19990126T090000,19990127T090000,19990128T090000,19990129T090000,19990130T090000,19990131T090000,20000101T090000,20000102T090000,20000103T090000,20000104T090000,20000105T090000,20000106T090000,20000107T090000,20000108T090000,20000109T090000,20000110T090000,20000111T090000,20000112T090000,20000113T090000,20000114T090000,20000115T090000,20000116T090000,20000117T090000,20000118T090000,20000119T090000,20000120T090000,20000121T090000,20000122T090000,20000123T090000,20000124T090000,20000125T090000,20000126T090000,20000127T090000,20000128T090000,20000129T090000,20000130T090000,20000131T090000
-
-RRULE:FREQ=DAILY;BYHOUR=9,10,11,12,13,14,15,16;BYMINUTE=0,20,40;COUNT=48
-DTSTART:19970902T090000
-INSTANCES:19970902T090000,19970902T092000,19970902T094000,19970902T100000,19970902T102000,19970902T104000,19970902T110000,19970902T112000,19970902T114000,19970902T120000,19970902T122000,19970902T124000,19970902T130000,19970902T132000,19970902T134000,19970902T140000,19970902T142000,19970902T144000,19970902T150000,19970902T152000,19970902T154000,19970902T160000,19970902T162000,19970902T164000,19970903T090000,19970903T092000,19970903T094000,19970903T100000,19970903T102000,19970903T104000,19970903T110000,19970903T112000,19970903T114000,19970903T120000,19970903T122000,19970903T124000,19970903T130000,19970903T132000,19970903T134000,19970903T140000,19970903T142000,19970903T144000,19970903T150000,19970903T152000,19970903T154000,19970903T160000,19970903T162000,19970903T164000
-
-RRULE:FREQ=HOURLY;INTERVAL=3;UNTIL=19970902T170000Z
-DTSTART:19970902T090000
-INSTANCES:19970902T090000,19970902T120000,19970902T150000
-
-RRULE:FREQ=MINUTELY;INTERVAL=15;COUNT=6
-DTSTART:19970902T090000
-INSTANCES:19970902T090000,19970902T091500,19970902T093000,19970902T094500,19970902T100000,19970902T101500
-
-RRULE:FREQ=MINUTELY;INTERVAL=90;COUNT=4
-DTSTART:19970902T090000
-INSTANCES:19970902T090000,19970902T103000,19970902T120000,19970902T133000
-
-RRULE:FREQ=DAILY;BYHOUR=9,10,11,12,13,14,15,16;BYMINUTE=0,20,40;COUNT=48
-DTSTART:19970902T090000
-INSTANCES:19970902T090000,19970902T092000,19970902T094000,19970902T100000,19970902T102000,19970902T104000,19970902T110000,19970902T112000,19970902T114000,19970902T120000,19970902T122000,19970902T124000,19970902T130000,19970902T132000,19970902T134000,19970902T140000,19970902T142000,19970902T144000,19970902T150000,19970902T152000,19970902T154000,19970902T160000,19970902T162000,19970902T164000,19970903T090000,19970903T092000,19970903T094000,19970903T100000,19970903T102000,19970903T104000,19970903T110000,19970903T112000,19970903T114000,19970903T120000,19970903T122000,19970903T124000,19970903T130000,19970903T132000,19970903T134000,19970903T140000,19970903T142000,19970903T144000,19970903T150000,19970903T152000,19970903T154000,19970903T160000,19970903T162000,19970903T164000
-
-RRULE:FREQ=WEEKLY;COUNT=35;BYDAY=SU,TH
-DTSTART:20111120T100000Z
-INSTANCES:20111120T100000Z,20111124T100000Z,20111127T100000Z,20111201T100000Z,20111204T100000Z,20111208T100000Z,20111211T100000Z,20111215T100000Z,20111218T100000Z,20111222T100000Z,20111225T100000Z,20111229T100000Z,20120101T100000Z,20120105T100000Z,20120108T100000Z,20120112T100000Z,20120115T100000Z,20120119T100000Z,20120122T100000Z,20120126T100000Z,20120129T100000Z,20120202T100000Z,20120205T100000Z,20120209T100000Z,20120212T100000Z,20120216T100000Z,20120219T100000Z,20120223T100000Z,20120226T100000Z,20120301T100000Z,20120304T100000Z,20120308T100000Z,20120311T100000Z,20120315T100000Z,20120318T100000Z
-
-RRULE:FREQ=DAILY;UNTIL=20141206T000000Z;BYMONTH=11,12,1,2,3,4,10
-DTSTART:20141030T000000Z
-INSTANCES:20141030T000000Z,20141031T000000Z,20141101T000000Z,20141102T000000Z,20141103T000000Z,20141104T000000Z,20141105T000000Z,20141106T000000Z,20141107T000000Z,20141108T000000Z,20141109T000000Z,20141110T000000Z,20141111T000000Z,20141112T000000Z,20141113T000000Z,20141114T000000Z,20141115T000000Z,20141116T000000Z,20141117T000000Z,20141118T000000Z,20141119T000000Z,20141120T000000Z,20141121T000000Z,20141122T000000Z,20141123T000000Z,20141124T000000Z,20141125T000000Z,20141126T000000Z,20141127T000000Z,20141128T000000Z,20141129T000000Z,20141130T000000Z,20141201T000000Z,20141202T000000Z,20141203T000000Z,20141204T000000Z,20141205T000000Z,20141206T000000Z
-
-RRULE:FREQ=YEARLY;BYWEEKNO=1,2;UNTIL=20170101T000000Z
-DTSTART:20130101T000000
-INSTANCES:20130101T000000,20130108T000000,20131231T000000,20140107T000000,20141230T000000,20150106T000000,20160105T000000,20160112T000000
-
-RRULE:FREQ=YEARLY;BYYEARDAY=366;UNTIL=20200101T000000
-DTSTART:20121231T120000
-INSTANCES:20121231T120000,20161231T120000
-
-RRULE:FREQ=YEARLY;BYDAY=-1FR;BYMONTH=10;UNTIL=20150101T000000Z
-DTSTART:20101029T120000
-INSTANCES:20101029T120000,20111028T120000,20121026T120000,20131025T120000,20141031T120000
-
-RRULE:FREQ=YEARLY;BYDAY=1FR;BYMONTH=4;UNTIL=20150101T000000Z
-DTSTART:20100402T120000
-INSTANCES:20100402T120000,20110401T120000,20120406T120000,20130405T120000,20140404T120000
-
-RRULE:FREQ=MONTHLY;BYMONTHDAY=31;COUNT=12
-DTSTART:20150131T000000Z
-INSTANCES:20150131T000000Z,20150331T000000Z,20150531T000000Z,20150731T000000Z,20150831T000000Z,20151031T000000Z,20151231T000000Z,20160131T000000Z,20160331T000000Z,20160531T000000Z,20160731T000000Z,20160831T000000Z
-
-RRULE:FREQ=MONTHLY;BYMONTHDAY=-31;COUNT=12
-DTSTART:20150101T000000Z
-INSTANCES:20150101T000000Z,20150301T000000Z,20150501T000000Z,20150701T000000Z,20150801T000000Z,20151001T000000Z,20151201T000000Z,20160101T000000Z,20160301T000000Z,20160501T000000Z,20160701T000000Z,20160801T000000Z
-
-RRULE:FREQ=YEARLY;BYYEARDAY=366;COUNT=3
-DTSTART:20121231T120000
-INSTANCES:20121231T120000,20161231T120000,20201231T120000
-
-RRULE:FREQ=YEARLY;BYYEARDAY=-366;COUNT=3
-DTSTART:20120101T120000
-INSTANCES:20120101T120000,20160101T120000,20200101T120000
-
-RRULE:FREQ=YEARLY;INTERVAL=2;UNTIL=20211231T140000Z;BYMONTH=2;BYDAY=WE
-DTSTART:20170201T090000
-START-AT:20200101T100000
-INSTANCES:20210203T090000,20210210T090000,20210217T090000,20210224T090000
-
-RRULE:FREQ=MONTHLY;UNTIL=20171229T000000Z;BYDAY=1FR
-DTSTART:20170901T090000
-START-AT:20170915T090000
-INSTANCES:20171006T090000,20171103T090000,20171201T090000
-
-RRULE:FREQ=WEEKLY;UNTIL=20170127T000000Z;WKST=MO;BYDAY=SU,TU,TH;INTERVAL=2
-DTSTART:20161229T090000
-START-AT:20161231T090000
-INSTANCES:20170101T090000,20170110T090000,20170112T090000,20170115T090000,20170124T090000,20170126T090000
-
-RRULE:FREQ=WEEKLY;UNTIL=20170127T000000Z;WKST=MO;BYDAY=SU,TU,TH;INTERVAL=2
-DTSTART:20161229T090000
-START-AT:20170102T090000
-INSTANCES:20170110T090000,20170112T090000,20170115T090000,20170124T090000,20170126T090000
-
-RRULE:FREQ=DAILY;UNTIL=20170131T140000Z;BYMONTH=1;INTERVAL=3
-DTSTART:20170101T090000
-START-AT:20170103T090000
-INSTANCES:20170104T090000,20170107T090000,20170110T090000,20170113T090000,20170116T090000,20170119T090000,20170122T090000,20170125T090000,20170128T090000,20170131T090000
-
-RRULE:FREQ=HOURLY;INTERVAL=3;UNTIL=20170903T170000Z
-DTSTART:20170902T090000
-START-AT:20170902T230000
-INSTANCES:20170903T000000,20170903T030000,20170903T060000,20170903T090000,20170903T120000,20170903T150000
-
-RRULE:FREQ=MONTHLY;INTERVAL=13;BYMONTH=5;COUNT=3
-DTSTART:20100212T000000
-INSTANCES:20130512T000000,20260512T000000,20390512T000000
-
-RRULE:FREQ=WEEKLY;BYDAY=WE,FR;INTERVAL=2;COUNT=4
-DTSTART:20190101T100000
-INSTANCES:20190102T100000,20190104T100000,20190116T100000,20190118T100000
-
-RRULE:FREQ=YEARLY;BYWEEKNO=1,2,-1,-2;BYDAY=TU;UNTIL=20170101T000000Z
-DTSTART:20130101T000000
-INSTANCES:20130101T000000,20130108T000000,20131217T000000,20131224T000000,20131231T000000,20140107T000000,20141216T000000,20141223T000000,20141230T000000,20150106T000000,20151222T000000,20151229T000000,20160105T000000,20160112T000000,20161220T000000,20161227T000000
-
-RRULE:FREQ=YEARLY;BYWEEKNO=53;BYDAY=TU,SA;UNTIL=20170101T000000Z
-DTSTART:20130101T000000
-INSTANCES:20151229T000000,20160102T000000
-
-RRULE:FREQ=HOURLY;UNTIL=20210303T000000Z
-DTSTART:20210301T142000
-START-AT:20210302T100000
-INSTANCES:20210302T102000,20210302T112000,20210302T122000,20210302T132000,20210302T142000,20210302T152000,20210302T162000,20210302T172000,20210302T182000,20210302T192000,20210302T202000,20210302T212000,20210302T222000,20210302T232000
```
</details>

**Comment by winterz** [comment]
this should be fixed in the 3.0 branch for the next 3.0.x release now.

**Comment by winterz** [comment]
however, still failing in the master branch.  re-opening

**Comment by winterz** [comment]
should be fixed in master and 3.0 branches now
